common/cultures/culture_vrykul.txt

```diff
@@ -10,24 +10,18 @@ vrykul_group = {
 
```

Comment:
History vrykul still use old names.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81067979-a339f080-8ef0-11ea-8a69-7551f8c1992c.png)

</details>
Also, the branch is super outdated.